server/src/test/java/org/candlepin/dto/api/v1/ConsumerDTOTest.java

```diff
@@ -92,6 +92,22 @@ public ConsumerDTOTest() {
 
         NestedOwnerDTO owner = new NestedOwnerDTO().id("OwnerId").displayName("Name").key("12345");
 
+        ContentDTO cDTO = new ContentDTO()
+            .id("ContentId")
+            .name("ContentName")
+            .modifiedProductIds(new HashSet<>(Arrays.asList("ID1", "ID2")));
+
+        Set<EnvironmentContentDTO> envContentDTO = new HashSet<>(Arrays.asList(new EnvironmentContentDTO()
```

Comment:
Nitpick: No need for Arrays.asList here if only 1 element is being added, you can create empty HashSet and then add it

Reply:
There's also ```Collections.singleton``` and ```Util.asSet```

Reply:
Done.